internal/actor/certificate_provider.go

```diff
@@ -22,6 +22,14 @@ const (
 	Online                                          // online
 )
 
+//go:generate enumerator -type CertificateProviderRelationshipLength -linecomment -trimprefix -empty
```

Comment:
you don't need `-trimprefix` as the values aren't prefixed

Reply:
Agh sorry, copy pasta.